DIRECTORY.md

```diff
@@ -836,7 +836,7 @@
   * [Daily Horoscope](https://github.com/TheAlgorithms/Python/blob/master/web_programming/daily_horoscope.py)
   * [Emails From Url](https://github.com/TheAlgorithms/Python/blob/master/web_programming/emails_from_url.py)
   * [Fetch Bbc News](https://github.com/TheAlgorithms/Python/blob/master/web_programming/fetch_bbc_news.py)
-  * [Fetch Github Info](https://github.com/TheAlgorithms/Python/blob/master/web_programming/fetch_github_info.py)
+  * [Fetch GitHub Info](https://github.com/TheAlgorithms/Python/blob/master/web_programming/fetch_github_info.py)
```

Comment:
I think you cannot modify this file.

Reply:
Correct, this file is updated automatically. So, please remove this change.